requirements.txt

```diff
@@ -0,0 +1,26 @@
+absl-py==0.8.0
+astor==0.8.0
+cloudpickle==1.2.1
+future==0.17.1
+gast==0.2.2
+google-pasta==0.1.7
+grpcio==1.23.0
+gym==0.14.0
+h5py==2.9.0
+Keras==2.2.4
+Keras-Applications==1.0.8
+Keras-Preprocessing==1.1.0
+Markdown==3.1.1
+numpy==1.17.1
+protobuf==3.9.1
+pyglet==1.3.2
+PyYAML==5.1.2
+-e git+https://github.com/HendrikPN/scigym.git@bf548ffb60e9c32f76af901cabdaf5a531d0a2ff#egg=scigym_pkg
```

Comment:
If [all] installs both the `requirements.txt` and the `setup.py` why do we need this line? (I just don't know the [all] comment so not sure how it works.)

Reply:
See my comment above about how [all] works. 

In summary, if we can use the [all] installation method for travis, then I don't actually think we need the `requirements.txt` file at all?